pkgs/development/python-modules/hypothesmith/default.nix

```diff
@@ -10,18 +10,31 @@ buildPythonPackage rec {
   };
 
   postPatch = ''
-    substituteInPlace setup.py --replace "lark-parser" "lark"
+    substituteInPlace setup.py \
+      --replace "lark-parser" "lark"
+
+    substituteInPlace tox.ini \
+      --replace "--cov=hypothesmith" "" \
+      --replace "--cov-branch" "" \
+      --replace "--cov-report=term-missing:skip-covered" "" \
+      --replace "--cov-fail-under=100" ""
   '';
 
   propagatedBuildInputs = [ hypothesis lark libcst ];
 
-  checkInputs = [ black parso pytestCheckHook pytest-cov pytest-xdist ];
+  checkInputs = [ parso pytestCheckHook pytest-xdist ];
 
   pytestFlagsArray = [
     "-v"
     "--numprocesses $NIX_BUILD_CORES"
   ];
```

Comment:
adding here `--no-cov` might remove the need to patch `tox.ini`, or it will only work when the package is actually present

Reply:
`--no-cov` only works if pytest-cov is installed

a some kind of stub for pytest would be nice

Reply:
How about we patch the code [here](https://github.com/pytest-dev/pytest-cov/blob/master/src/pytest_cov/plugin.py#L183=) to check for an env or something else and then exit?